packages/-ember-data/tests/unit/model/rollback-attributes-test.js

```diff
@@ -367,7 +368,11 @@ module('unit/model/rollbackAttributes - model.rollbackAttributes()', function(ho
   });
 
   test("invalid record's attributes can be rollbacked", async function(assert) {
-    assert.expect(12);
+    if (gte('3.13.0')) {
```

Comment:
I think these should be `3.13.0-beta.1`, right? Otherwise it will continue to fail until 3.13.0 is released as stable.

Reply:
Gte treats beta as satisfying this